sova/cli/commands/maintain.py

```
@@ -0,0 +1,129 @@
+from typing import Annotated, Optional
+
+import typer
+from rich.console import Console
+from rich.table import Table
+
+console = Console(stderr=True)
+
+
+def maintain(
+    project: Annotated[Optional[Path], typer.Option("--project", "-p", help="Project directory.")] = None,
+    dry_run: Annotated[bool, typer.Option("--dry-run", help="Show what would happen without merging.")] = False,
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)